main.md

```diff
@@ -2,27 +2,27 @@
 
-**tl;dr? skip the intro** and [get started](guide/install.md)!
+**tl;dr? Skip the intro** and [get started](guide/install.md)!
 
-**Don't like to reading?** Watch the [Testnet Video Guide](#) instead.
+**Don't like to read?** Watch the [Testnet Video Guide](#) instead.
 
 ## What is Spacemesh?
-Spacemesh is a programmable cryptocurrency open-source platform powered by a novel proof-of-space-time (PoST) consensus protocol. Spacemesh does not use  proof of stake or proof of work protocols, thus it avoids some of their inherent issues, such as massive energy waste, mining pooling and centralization.
+Spacemesh is a programmable cryptocurrency open-source platform powered by a novel proof-of-space-time (PoST) consensus protocol. Spacemesh does not use proof-of-stake or proof-of-work (PoW) protocols, thus it avoids some of their inherent issues, such as massive energy waste, mining pooling and centralization.
 
 ## What does Spacemesh try to achieve?
 Spacemesh  is designed to achieve three main objectives:
 1. Create and maintain the ledger for the `Spacemesh Coin` - a fair cryptocurrency that will become a global means of payment.
 2. Ensure fair distribution of the Spacemesh Coin cryptocurrency.
 3. Establish a global programmable infrastructure for tokenized apps, communities and economies.
 
-## What problems does Spacemesh plan to solve?
+## What problems does Spacemesh plan to solve and how?
 1. Problem: Mining cryptocurrency today can only be done by a select few. Solution: We aim to create a world in which any interested party can mine Spacemesh cryptocurrency, which will naturally be followed by a fair distribution of coin (i.e., `The People's Coin`).
 2. Problem: Energy consumption due to PoW chains is skyrocketing. Solution: Spacemesh has improved upon PoW by creating a consensus protocol that consumes significantly less energy (i.e., `A Better Bitcoin`).
-3. Problem: Lack of satisfactory smart contracts infrastructure for creation of fair communities and decentralized financial services. Solution: we plan to add a modern smart contracts virtual machine (SVM), a new modern smart contracts language (Smesh) and dev tools designed for developers to build applications on the Spacemesh platform.
+3. Problem: Lack of satisfactory smart contracts infrastructure for creation of fair communities and decentralized financial services. Solution: we plan to add a modern smart contracts virtual machine, called Spacemesh virtual machine (SVM), a new modern smart contracts language (Smesh) and dev tools designed for developers to build applications on the Spacemesh platform.
 
 ## What makes Spacemesh unique?
 A mesh topology combined with a novel consensus PoST protocol that allows anyone to join the decentralized Spacemesh blockchain computer without requiring permission, simply by committing some free hard-drive space.
```

Comment:
What does exactly a ``blockchain computer" mean?

Reply:
A smart contracts executor with smart contracts storage running on every full node. The computer is made from all nodes executing smart contracts to update and maintain the same state.